lib/commands/init.js

```diff
@@ -77,24 +77,35 @@ function saveJson(project, logger, json) {
 
 function setDefaults(config, json) {
     var name;
+    var npm = {};
     var promise = Q.resolve();
 
+    // Grab values from package.json if it exists
+    if (fs.existsSync(path.join(config.cwd, 'package.json'))) {
```

Comment:
Dont use `fs.existsSync` as it will be deprecated. Just `try` reading the file and catch any error.